### PR TITLE
added drink carousel heading and increased to 12 items in carousel

### PR DESCRIPTION
**Pull request by yardboy2401**

### CODE REVIEW

File: assets/js/scripts.js [hunk 1]
```diff
@@ -104,8 +104,11 @@ $(document).ready(function () {
     $("#item3").empty();
     $("#item4").empty();
     $("#item5").empty();
+    $("#drinkHeading").empty();
     let drinkArray = ["one", "two", "three", "four", "five"];
```
Comment (bravotango) [inline assets/js/scripts.js:108]: Is drinkArray used?

File: index.html [hunk 2]
```diff
@@ -32,14 +32,21 @@ <h5>Search History</h5>
...
               <a id="item4" class="carousel-item"></a>
               <a id="item5" class="carousel-item"></a>
+              <a id="item6" class="carousel-item"></a>
+              <a id="item7" class="carousel-item"></a>
+              <a id="item8" class="carousel-item"></a>
+              <a id="item9" class="carousel-item"></a>
+              <a id="item10" class="carousel-item"></a>
+              <a id="item11" class="carousel-item"></a>
+              <a id="item12" class="carousel-item"></a>
             </div>
```
Comment (bravotango) [inline index.html:50]: I think you already tried, but lines 38-49 look like they should be in the JavaScript file

**Review by bravotango — APPROVED** [review]

File: assets/js/scripts.js (hunk 1 above)
Comment (yardboy2401) [reply assets/js/scripts.js]: no it's not. I just removed and pushed. Sorry I noticed that earlier and forgot to delete it then.

File: index.html (hunk 2 above)
Comment (yardboy2401) [reply index.html:50]: Yep I added .empty() for those as well. For loop is good for 12 items now.

**Pull request merged**